src/ngx-gallery.component.ts

```diff
@@ -268,9 +268,14 @@ export class NgxGalleryComponent implements OnInit, DoCheck, AfterViewInit   {
     }
 
     private setImages(): void {
+        const _this:NgxGalleryComponent = this;
```

Comment:
Could be avoided with fat arrow functions.

Reply:
You are write. I replaced it with arrow function.